### ISSUE TITLE
Config of hyper-parameters

### ISSUE THREAD

**Issue by avlrd** [avlrd/projetIA#10]
Reach best perfs, close_mosaic = 0 and seed 42 (check guide to fine-tune parameters)
